docs/daemon/configuration/genesis.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 3
+sidebar_label: "Genesis"
+id: daemon-genesis
+title: Genesis Configuration
+description: Genesis specification
+slug: /daemon/config/genesis
+---
+
+A genesis file defines the initial state and parameters needed to bootstrap a new blockchain network. It serves as the starting point for the entire blockchain system by defining key aspects of the network, including its initial validators, initial height, consensus rules, and more. All the nodes in the network need to have the same genesis file to start with the same initial conditions. Any discrepancies or errors in the genesis file could prohibit the node from joining the network or producing blocks.
+
+## Minimal Configuration
+
+Below is a sample `genesis.json` file featuring a basic configuration:
+
+```json title="sample genesis.json file"
+{
+  "genesis_time": "2023-08-28T22:24:04.513472Z",
+  "chain_id": "kwil-chain-RQMl2F",
+  "initial_height": "0",
+  "validators": [
+    {
+      "pub_key": "71d79c64ffb46e8f6c36f1429066a9d92f6adf5bbe546204e40ec5559478b6aa",
+      "power": 1,
+      "name": "validator-0"
+    }
+  ],
+  "app_hash": "4F8735846B8CFB01A277430A392DD3DB7B66510C358C5F4A2AF88A348CB20A62"
+}
+```
+
+## Genesis File Contents
+
+### Genesis Time
+
+ `genesis_time`: This field specifies the official time (UTC format) at which the blockchain is initialized.
```

Comment:
UTC is a time standard, not a format.

Reply:
It can be represented as RF 2822, ISO 8601, human-readable, etc.  I'm more looking at what cometbft requires wrt those

Reply:
Yeah, Its ISO 8601, updated it